lib/cc/engine/file_list_resolver.rb

```diff
@@ -29,13 +29,13 @@ def exclude_due_to_config?(path)
       end
 
       def include_based_files_to_inspect
-        @include_paths.map do |path|
-          if path =~ %r{/$}
+        @include_paths.map(&method(:abs_path)).flat_map do |path|
```

Comment:
@jpignata not totally clear what you mean. Were you thinking something like `@include_paths.flat_map(&method(:expanded_include_path)).compact` as a final form?